cmd/influxd/run.go

```diff
@@ -50,9 +50,14 @@ func Run(config *Config, join, version string, logWriter *os.File) (*messaging.B
 		// We want to make sure we are spun up before we exit this function, so we manually listen and serve
 		listener, err := net.Listen("tcp", config.BrokerAddr())
 		if err != nil {
-			log.Fatal(err)
+			log.Fatalf("failed to listen for broker on tcp: %s ", err)
 		}
-		go func() { log.Fatal(http.Serve(listener, h)) }()
+		go func() {
+			err := http.Serve(listener, h)
+			if err != nil {
+				log.Fatalf("failed to serve broker: %s", err)
```

Comment:
I would be a little clearer here too. Why not something like "Broker HTTP server failed to start: %s"?